here-naksha-lib-psql/src/commonMain/kotlin/naksha/psql/executors/query/WhereClauseBuilder.kt

```diff
@@ -115,6 +120,16 @@ class WhereClauseBuilder(private val request: ReadFeatures) {
         }
     }
 
+    private fun resolveTransformation(transformation: SpTransformation, geometryStmt: String): String {
+        return when(transformation){
+            is SpBuffer -> "ST_Buffer($geometryStmt, ${transformation.distance})"
```

Comment:
we need geography transformation as well, default st_buffer will use distance in radius, and we need meters, which are available only for geography

Reply:
Understood - initially I only wanted basic buffer to work (not knowing about meters being disregarded).
I will support this fully

Reply:
done